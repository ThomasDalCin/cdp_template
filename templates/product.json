/*
 * ------------------------------------------------------------
 * IMPORTANT: The contents of this file are auto-generated.
 *
 * This file may be updated by the Shopify admin theme editor
 * or related systems. Please exercise caution as any changes
 * made to this file may be overwritten.
 * ------------------------------------------------------------
 */
{
  "sections": {
    "gallery-pdp": {
      "type": "gallery-pdp",
      "settings": {}
    },
    "info-pdp": {
      "type": "info-pdp",
      "settings": {
        "cta_delear": "Trova un dealer",
        "cta_brochure": "Scarica la brochure"
      }
    },
    "marketing-pop-up": {
      "type": "marketing-pop-up",
      "settings": {
        "marketing_pop_up_title": "Consent to direct marketing",
        "marketing_pop_up_content": "By selecting this box you give your consent to receive commercial communications and to participate in market research and activities to measure your level of satisfaction, regarding Pardo Yachts products and services, via e-mail and/or SMS, telephone and traditional paper mail. For this purpose we will use the personal and contact data that you have communicated to us (e.g. name, surname, address, e-mail, mobile number, etc.). The data collected will be stored until a request to withdraw consent is received from you. The Data Controller is Cantiere Del Pardo SpA. By writing to data.protection@calzedonia.com you can exercise the right: i) to access, to find out if your data is being processed and to obtain a copy; ii) to obtain the rectification or integration of your data; iii) to request the deletion of your data; iv) to obtain the limitation of processing; v) data portability; vi) revocation of consent. You can also lodge a complaint with the competent data protection supervisory authority if you believe that the processing of your personal data is in conflict with the provisions of EU Regulation 2016/679. Complete information: Privacy Policy."
      }
    },
    "profilation-pop-up": {
      "type": "profilation-pop-up",
      "settings": {
        "profilation_pop_up_title": "Consent to direct profilation",
        "profilation_pop_up_content": "By checking this box, we will send personalized (profiled) commercial communications. For this purpose, we will use the data relating to the «Pardo Yachts» boats that you have purchased or for which you have requested information, your country of origin, your personal data, your interactions with us through our website, through the Apps - which may be developed by us or by third parties - and through our social channels: we will thus send you personalized communications and we will be able to develop products and services consistent with the preferences of our customers. Furthermore, your profile may be enriched with information acquired lawfully from third parties and made available to Cantiere del Pardo. The data will be kept for 5 years from the date of collection. The Data Controller is Cantiere del Pardo Spa. By writing to data.protection@calzedonia.com you can exercise the right: i) to access, to find out if your data is being processed and to obtain a copy; ii) to obtain the rectification or integration of your data; iii) to request the deletion of your data; iv) to obtain the limitation of processing; v) data portability; vi) revocation of consent. You can also lodge a complaint with the competent data protection supervisory authority if you believe that the processing of your personal data is in conflict with the provisions of EU Regulation 2016/679. Complete information: Privacy Policy."
      }
    },
    "ext-int-pdp": {
      "type": "ext-int-pdp",
      "settings": {
        "main_title": "Design",
        "exterior_text_title": "Exteriors",
        "cta_brochure": "Full exterior gallery",
        "interior_text_title": "Interiors",
        "cta_brochure_interior": "Full interior gallery",
        "cta_content_text": "Read more"
      }
    },
    "top-view-pdp": {
      "type": "top-view-pdp",
      "settings": {
        "main_title": "Top view",
        "standard_badge_text": "Standard",
        "optional_badge_text": "Optional",
        "see_more_images": "Full View"
      }
    },
    "features-pdp": {
      "type": "features-pdp",
      "settings": {
        "product_basics": "Project",
        "basics_dropdown_title": "Basic Boat Details",
        "technical_features": "Technical Feature",
        "dimensions_dropdown_title": "Dimensions",
        "autonomy_dropdown_title": "Autonomy",
        "engine_dropdown_title": "Engine",
        "accommodations_dropdown_title": "Accommodations",
        "general_features": "General arrangement"
      }
    },
    "overview-pdp": {
      "type": "overview-pdp",
      "settings": {}
    },
    "find-delear": {
      "type": "find-delear",
      "custom_css": [
        "@media (min-width: 1024px) {.lg\\:min-h-\\[460px\\] {min-height: 55vh; }}"
      ],
      "settings": {
        "section_title": "È ora di trovare un dealer",
        "section_image": "shopify://shop_images/Pardo-Yachts-Find-a-dealer_69ecd90a-1c13-4492-9582-ccf862f1096c.jpg",
        "section_text": "L’esperienza dei nostri esperti dealers è insostituibile. Trova il dealer più vicino a te per saperne di più ed entrare nel mondo Pardo Yacht",
        "cta_text": "Trova un dealer"
      }
    }
  },
  "order": [
    "gallery-pdp",
    "info-pdp",
    "marketing-pop-up",
    "profilation-pop-up",
    "ext-int-pdp",
    "top-view-pdp",
    "features-pdp",
    "overview-pdp",
    "find-delear"
  ]
}
