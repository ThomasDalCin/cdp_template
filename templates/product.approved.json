{
    "sections": {
        "top-approved-details": {
            "type": "top-approved-details",
            "settings": {}
        },
        "product-approved-features": {
            "type": "product-approved-features",
            "settings": {}
        }
    },
    "order": [
     "top-approved-details",
     "product-approved-features"
    ]
  }
  